app/src/context/Exchange.tsx

```diff
@@ -162,7 +162,11 @@ export const ExchangeProvider: React.FC<{
       }
     } catch (error: any) {
       console.log(error);
-      Alert.alert(error.response.data);
+      if (error.response.data === "Username or password is incorrect") {
```

Comment:
I thought i said the stand up that this error deoesnt make sense as the username / email HAS to be correct.
```suggestion
      if (error.response.data === "The password you have provided for this account is incorrect") {
```